packages/@expo/cli/src/start/server/middleware/InterstitialPageMiddleware.ts

```diff
@@ -75,7 +76,7 @@ export class InterstitialPageMiddleware extends ExpoMiddleware {
     res = disableResponseCache(res);
     res.setHeader('Content-Type', 'text/html');
 
-    const platform = parsePlatformHeader(req);
+    const platform = parsePlatformHeader(req) ?? resolvePlatformFromUserAgentHeader(req);
```

Comment:
This method is attempting to implement our [hosted server's specification](https://docs.expo.dev/technical-specs/expo-updates-0/#manifest-request) with added legacy support for expo-cli which supported the `exponent-platform` header, is platform sniffing worked into the specification anywhere?